src/Sfa.Tl.ResultsAndCertification.Web/ViewModel/AdminDashboard/IndustryPlacement/AdminReviewChangesIndustryPlacementViewModel.cs

```diff
@@ -0,0 +1,159 @@
+﻿using Sfa.Tl.ResultsAndCertification.Common.Helpers;
+using Sfa.Tl.ResultsAndCertification.Web.Content.AdminDashboard;
+using Sfa.Tl.ResultsAndCertification.Web.Utilities.CustomValidations;
+using Sfa.Tl.ResultsAndCertification.Web.ViewComponents.BackLink;
+using Sfa.Tl.ResultsAndCertification.Web.ViewComponents.Summary.SummaryItem;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using IpStatus = Sfa.Tl.ResultsAndCertification.Common.Enum.IndustryPlacementStatus;
+
+namespace Sfa.Tl.ResultsAndCertification.Web.ViewModel.AdminDashboard.IndustryPlacement
+{
+    public class AdminReviewChangesIndustryPlacementViewModel
+    {
+        public int RegistrationPathwayId { get; set; }
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
+        public long Uln { get; set; }
+        public string ProviderName { get; set; }
+        public int ProviderUkprn { get; set; }
+        public string TlevelName { get; set; }
+        public string Learner => $"{FirstName} {LastName}";
+        public AdminChangeIpViewModel AdminChangeIpViewModel { get; set; }
+
+        [Required(ErrorMessageResourceType = typeof(ReviewChangesIndustryPlacement), ErrorMessageResourceName = "Validation_Contact_Name_Blank_Text")]
+        public string ContactName { get; set; }
```

Comment:
Same with the review changes page properties to show at the bottom... day, month, zendesk ID etc